benchmarks/maxtext_trillium_model_configs.py

```diff
@@ -320,6 +359,45 @@ class MaxTextModel:
     ),
 )
 
+llama2_70b_4096_pw_long_run = MaxTextModel(
+    model_name="llama2-70b-4096-pw-lr",
```

Comment:
benchmark name can be `llama2-70b-4096-checkpoint`?

Reply:
We don't want the name to be too long because it may overrun the 40 character limit for XPK :(